web/frontend/.last_build_id

```diff
@@ -0,0 +1 @@
+52b80473a43ce48203e646c64a29d6b0
```

Comment:
do we need to commit this file? Additionally there are some `*.pyc` and `*.ttf` files which i'm not sure you need to commit either. Correct me if i'm wrong but otherwise add this stuff to the .gitignore : ).